src/transformers/pipelines/token_classification.py

```diff
@@ -228,6 +235,59 @@ def __call__(self, inputs: Union[str, List[str]], **kwargs):
             return answers[0]
         return answers
 
+    def set_subwords_label(self, entities: List[dict], strategy: str) -> dict:
+
+        def sub_words_label(sub_words: List[dict]) -> dict:
+            score = np.stack([sub["score"] for sub in sub_words])
+            if strategy == "default":
```

Comment:
as @joshdevins said: "If training with padded sub-words/label for first sub-word only, e.g. Max Mustermann → Max Must ##erman ##n → B-PER I-PER X X
Use the label from the first sub-word (default)"